# Margin ratio of grid map
grid_map_margin_ratio: 0.5

# Resolution of grid map [m]
grid_map_resolution: 0.02

# Height scale of grid map
grid_map_height_scale: 0.02

# Theta threshold for slicing SE2 sample [deg]
slice_se2_theta_thre: 2.5

# Z threshold for slicing R3 sample [m]
slice_r3_z_thre: 0.025

# Z threshold for slicing SE3 sample [m]
slice_se3_z_thre: 0.1

# Theta threshold for slicing SE3 sample [deg]
slice_se3_theta_thre: 20

# ROS bag path of sample set for evaluation
eval_bag_path: "" # Set by rosparam since it contains ROS package path

# SVM threshold list for evaluation
eval_svm_thre_list: [-1.        , -0.97979798, -0.95959596, -0.93939394, -0.91919192,
       -0.8989899 , -0.87878788, -0.85858586, -0.83838384, -0.81818182,
       -0.7979798 , -0.77777778, -0.75757576, -0.73737374, -0.71717172,
       -0.6969697 , -0.67676768, -0.65656566, -0.63636364, -0.61616162,
       -0.5959596 , -0.57575758, -0.55555556, -0.53535354, -0.51515152,
       -0.49494949, -0.47474747, -0.45454545, -0.43434343, -0.41414141,
       -0.39393939, -0.37373737, -0.35353535, -0.33333333, -0.31313131,
       -0.29292929, -0.27272727, -0.25252525, -0.23232323, -0.21212121,
       -0.19191919, -0.17171717, -0.15151515, -0.13131313, -0.11111111,
       -0.09090909, -0.07070707, -0.05050505, -0.03030303, -0.01010101,
        0.01010101,  0.03030303,  0.05050505,  0.07070707,  0.09090909,
        0.11111111,  0.13131313,  0.15151515,  0.17171717,  0.19191919,
        0.21212121,  0.23232323,  0.25252525,  0.27272727,  0.29292929,
        0.31313131,  0.33333333,  0.35353535,  0.37373737,  0.39393939,
        0.41414141,  0.43434343,  0.45454545,  0.47474747,  0.49494949,
        0.51515152,  0.53535354,  0.55555556,  0.57575758,  0.5959596 ,
        0.61616162,  0.63636364,  0.65656566,  0.67676768,  0.6969697 ,
        0.71717172,  0.73737374,  0.75757576,  0.77777778,  0.7979798 ,
        0.81818182,  0.83838384,  0.85858586,  0.87878788,  0.8989899 ,
        0.91919192,  0.93939394,  0.95959596,  0.97979798,  1.        ]

# Threshold list of distaice ratio for evaluation by one-class nearest neighbor
ocnn_dist_ratio_thre_list: [1.0, 2.0, 3.0, 4.0, 5.0, 6.0]

# K (i.e., number of nearest samples) list for evaluation by k-nearest neighbor
knn_K_list: [1, 3, 5, 7, 9]

# Height of xy plane marker
svm_thre: -0.1

# SVM gamma (smoothness. smaller is smoother)
svm_gamma: 30

# SVM nu (outliers ratio)
svm_nu: 0.05
